lib/twilio-ruby/version.rb

```diff
@@ -1,3 +1,9 @@
 module Twilio
-    VERSION = '5.4.2'
```

Comment:
I think this will break our release process as it auto-populates this value. I'd suggest leaving VERSION the same and adding `MAJOR`, `MINOR`, `TINY`/`PATCH` by parsing the relevant part from `VERSION`